### DISCUSSION TITLE
I love love love ADS

### DISCUSSION THREAD

**Discussion by ishazel** [microsoft/azuredatastudio#23431] [General]
Just wanted to say THANK YOU to everyone who has contributed! It makes my work so much more pleasant and enjoyable. All the recent updates have been super welcome 👏🏻👏🏻👏🏻

**Comment by erinstellato-ms** [comment]
@ishazel thanks so much for your feedback!  We are working hard to improve Azure Data Studio in terms of the functionality it offers and overall stability.  Stay tuned :)
